src/routes/resources/[resourceContentId]/+page.svelte

```diff
@@ -662,7 +662,7 @@
                                 <div class="me-2 font-semibold text-gray-700">Draft</div>
                             {/if}
                         </div>
-                        <div class="w-full flex-grow">
+                        <div class="mt-[0.9375rem] w-full flex-grow">
```

Comment:
@kaseywright I like using rem here for this, I'm realizing now we're doing calcs in different places with pixels instead of rems. For instance above we have `h-[calc(100vh-245px)]`. May be worth at some point doing a refactor of those to convert them to rems.